system/ee/ExpressionEngine/Addons/filepicker/Service/FilePicker/Factory.php

```diff
@@ -53,26 +53,6 @@ public function make($dirs = 'all')
 
         return $fp;
     }
-
-    /**
-     * Handle a filepicker request. Does all the default stuff.
```

Comment:
@intoeetive I'm wondering why these were removed?

Reply:
These were never called and also referenced non-existing class `Endpoint`

Reply:
I guess my concern with this one is if a third party is using it?  (thinking handle request more then from request that's using a class that doesn't exist.)

Thoughts @intoeetive

Reply:
I don't believe the handleRequest method was referencing a non-existing class?

FromRequest was though?

Reply:
`handleRequest` is calling `$this->fromRequest()` which calls non-existing `Endpoint`